components/Header/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Image from "next/image";
+import { Link as ScrollLink } from "react-scroll";
+import { HideAt, ShowAt } from "react-with-breakpoints";
+
+import MobileNavbar from "./MobileNavbar";
+import { headerNavbarLinks } from "./constants";
+
+const Header: React.FC = () => (
+  <header className="bg-black w-screen fixed flex items-center justify-between py-4 px-8">
+    <div className="flex items-center w-full justify-between max-w-screen-lg mx-auto">
+      <Image
+        className="md:h-20 h-10"
+        loading="eager"
+        quality={100}
+        title="Floripa+"
+        src="/images/logo.png"
+        alt="Floripa Mais Logo"
+        unsized
```

Comment:
I had some issues in order to set a fixed width and height different for different devices widths 

Still need to see if it's the best solution because I'm afraid that using ``unsized`` will result in the content shifting on the first load. 

Maybe would be better to test the images loading on a slow network.

Reply:
Would be better to use the ``sizes`` property and then set the height according to a ``max-width`` but it isn't working - Refer to https://github.com/vercel/next.js/issues/18413 

And implemented another solution as a workaround https://github.com/vercel/next.js/issues/18413#issuecomment-721739533